libs/sdk-ui-kit/src/Dropdown/Dropdown.tsx

```diff
@@ -207,6 +216,7 @@ export const Dropdown: React.FC<IDropdownProps> = (props) => {
                 closeOnOutsideClick={closeOnOutsideClick}
                 closeOnMouseDrag={closeOnMouseDrag}
                 closeOnParentScroll={closeOnParentScroll}
+                closeOnEscape
```

Comment:
Does it work correctly when there's dropdown in the modal dialog? Does the focus trap handle it?

Reply:
Not handled automatically now. If DD is in Modal, then Modal creator needs to stop propagation of Esc from DD to modal manually